autoload/neoformat/formatters/rust.vim

```diff
@@ -3,8 +3,12 @@ function! neoformat#formatters#rust#enabled() abort
 endfunction
 
 function! neoformat#formatters#rust#rustfmt() abort
+    let opts = get(g:, 'rustfmt_edition_opt', '')
+    if opts != ''
+        let opts = '--edition ' . opts
```

Comment:
does this require an `endif`?

Reply:
Yes, you are right, the endif has been added.